profiles/api_test.py

```diff
@@ -33,24 +27,8 @@ def test_ensure_profile(mocker, profile_data, no_profile):
         if profile_data and 'image' in profile_data:
             assert profile.image == profile_data['image']
         else:
-            base_img_url = 'https://www.gravatar.com/avatar/7ec7606c46a14a7ef514d1f1f9038823.jpg'
-            assert profile.image == base_img_url
-            assert profile.image_small == '{}?s={}'.format(base_img_url, IMAGE_SMALL_MAX_DIMENSION)
-            assert profile.image_medium == '{}?s={}'.format(base_img_url, IMAGE_MEDIUM_MAX_DIMENSION)
+            assert not profile.image
     else:
         assert 'gravatar.com' not in profile.image
         assert 'gravatar.com' not in profile.image_small
         assert 'gravatar.com' not in profile.image_medium
```

Comment:
Do you know why this testing logic is still necessary (or ever was)? From what I can tell, this is just checking that some fields generated by our `ProfileFactory` don't have the string 'gravatar.com' in them.